src/packages/src/xroad/redhat/SPECS/xroad-securityserver-fi.spec

```diff
@@ -37,3 +37,7 @@ cp -p %{srcdir}/default-configuration/override-securityserver-fi.ini %{buildroot
 
 %post
 
+%postun
+if [ $1 -gt 0 ] ; then
```

Comment:
redundant test, the macro already checks for upgrade

Reply:
So it seems, yeah. Removed the checks from all spec-files.